sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/bigtable/BigtableIOTest.java

```diff
@@ -176,7 +176,7 @@ public void testReadBuildsCorrectly() {
   public void testReadValidationFailsMissingTable() {
     BigtableIO.Read read = BigtableIO.read().withBigtableOptions(BIGTABLE_OPTIONS);
 
-    thrown.expect(IllegalArgumentException.class);
+    thrown.expect(NullPointerException.class);
```

Comment:
This should really still continue to throw the IllegalArgumentException

How do you know the correct exception is being thrown since NPE/IllegalArgument could refer to many incorrect parameters?

Reply:
ok